tests/integration/eventHistories.test.js

```diff
@@ -91,24 +91,24 @@ describe('GET /eventhistories', () => {
     expect(response.statusCode).toEqual(400);
   });
 
-  it('should return a 403 if at least one auxiliary is not from the same company', async () => {
+  it('should return a 404 if at least one auxiliary is not from the same company', async () => {
```

Comment:
je pense que le test au dessus ('should return a 400 if invalid query') est contenu dans ce test et qu'il n'est donc pas utile

Reply:
tu parles de "should return a 400 if invalid action in query" ? lui il est utile non ?

Reply:
non je parle de 'should return a 400 if invalid query ligne 93 où l'auxiliaire passé en query n'en ai pas un